src/docs/references/extended-data/index.md

```diff
@@ -86,12 +79,13 @@ schema is provided for a given extended data key, the API can use this
 information to make querying the extended data possible via some API
 endpoints. For instance
 [/listings/query](https://www.sharetribe.com/api-reference/marketplace.html#query-listings)
-supports querying listings by public data.
+supports querying listings by public data or metadata.
 
 Search schemas can be managed by
 [Flex CLI](/flex-cli/getting-started-with-flex-cli/). With the CLI, you
 can list, set and unset search schemas to listing's Public data and
```

Comment:
```suggestion
can list, set and unset search schemas to listing's public data and
```

Reply:
@bladealslayer Original documentation used capitalized words, any reason to change it to lower-cased?

Reply:
https://github.com/sharetribe/flex-docs/pull/509/files/0bb1faf93d421059be6e19b86fa7fb0ceec03189#diff-99a323a1d00270c684ecfe9699377d8861f80ad746edd607bf72db5f978f3680R62-R63

Line 62 - 63 should be lowercased as well?

Reply:
Yeah, I think they were capitalized by mistake and shouldn't have been. "Metadata" on line 63 is the first word in the sentence so it can stay capitalized.